File: csharp/Behavioral/14-Command/Model/JobCard.cs
public class JobCard
{
    public string  JobId    { get; init; } = Guid.NewGuid().ToString()[..8];
    public string  Title    { get; init; } = string.Empty;
    public string  Category { get; init; } = string.Empty;
    public string  Status   { get; set;  } = "Draft";
    public string? Assignee { get; set;  }
    public override string ToString() =>
        $"Job #{JobId} '{Title}' [{Category}] — Status: {Status}" +
        (Assignee is not null ? $" | Assignee: {Assignee}" : "");
}

File: typescript/Behavioral/14-Command/Model/JobCard.ts
export class JobCard {
  jobId: string = Math.random().toString(36).substring(2, 10);
  title: string = "";
  category: string = "";
  status: string = "Draft";
  assignee: string | null = null;

  toString(): string {
    return (
      `Job #${this.jobId} '${this.title}' [${this.category}] — Status: ${this.status}` +
      (this.assignee !== null ? ` | Assignee: ${this.assignee}` : "")
    );
  }
}
